js/form-validate.js

```diff
@@ -0,0 +1,79 @@
+const Capacity = {
+  '1': ['1'],
+  '2': ['1', '2'],
+  '3': ['1', '2', '3'],
+  '100': ['0'],
```

Comment:
В объекте ключи всегда строка, нет необходимости явно выделять в строку
```suggestion
  1: ['1'],
  2: ['1', '2'],
  3: ['1', '2', '3'],
  100: ['0'],
```